LinkifiedText/index.jsx

```diff
@@ -4,15 +4,17 @@ import Text from '../Text';
 import Link from '../Link';
 import colors from '../style/color';
 
-const linkDataToElement = (options, link, key) =>
+const linkDataToElement = (options, link, key) => (
   <Link
     href={link.url}
     key={key}
     unstyled={options.unstyled}
     newTab={options.newTab}
+    className={link.className ? link.className : ''}
```

Comment:
I think we can simplify this:

```suggestion
    className={link.className}
```